adoptingerlang.org

```diff
@@ -1923,18 +1923,19 @@ $ docker-compose up
 #+attr_shortcode: info
 #+attr_latex: :environment bclogo :options [logo=\bcinfo, couleurBarre=orange, noborder=true, couleur=white]{Information}
 #+begin_notice
-You don't have to start everything to get an Erlang shell from your release. Every release built with <code>rebar3</code> comes with a boot script called <code>start_clean</code> which only starts the <code>kernel</code> and <code>stdlib</code> applications. This can be run with the command <code>console_clean</code> and can be useful for debugging purposes.
+You don't have to start everything to get an Erlang shell from your release. Every release built with <code>rebar3</code> comes with a boot script called <code>start&#95;clean</code> which only starts the <code>kernel</code> and <code>stdlib</code> applications. This can be run with the command <code>console&#95;clean</code> and can be useful for debugging purposes.
 #+end_notice
 
 To boot the development release to an interactive Erlang shell run the extended start script with command =console=:
 
 #+BEGIN_SRC shell
-$ DB_HOST=localhost LOGGER_LEVEL=debug SCHEDULERS=1 \
-  _build/default/rel/service_discovery/bin/service_discovery console
+$ _build/default/rel/service_discovery/bin/service_discovery console
+Erlang/OTP 21 [erts-10.4.4] [source] [64-bit] [smp:1:1] [ds:1:1:10] [async-threads:30] [hipe]
```

Comment:
I think you had built the release on OTP-22 a bit earlier. Not a big deal, just felt like pointing it out; we can go ahead without bothering to chagne this

Reply:
Ah no, let me fix those. Probably some other places too.